pkg/util/typeutil/schema.go

```diff
@@ -1466,7 +1466,11 @@ func ValidateSparseFloatRows(rows ...[]byte) error {
 			return fmt.Errorf("invalid data length in sparse float vector: %d", len(row))
 		}
 		for i := 0; i < SparseFloatRowElementCount(row); i++ {
-			if i > 0 && SparseFloatRowIndexAt(row, i) < SparseFloatRowIndexAt(row, i-1) {
+			idx := SparseFloatRowIndexAt(row, i)
+			if idx == math.MaxUint32 {
+				return errors.New("invalid index in sparse float vector: must be less than 2^32-1")
```

Comment:
maybe use `merr.WrapErrInvalidParameterMsg("invalid index in sparse float vector: must be less than 2^32-1")`

Reply:
can't for now, import cycle found if we import `merr` here. maybe fix later
```
package github.com/milvus-io/milvus/pkg/util/typeutil
	imports github.com/milvus-io/milvus/pkg/util/merr
	imports github.com/milvus-io/milvus/pkg/util/paramtable
	imports github.com/milvus-io/milvus/pkg/config
	imports github.com/milvus-io/milvus/pkg/util/typeutil: import cycle not allowed
```